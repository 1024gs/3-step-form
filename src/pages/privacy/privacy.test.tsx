import React from '../../../vendors/react';
import {shallow} from 'enzyme';
import PrivacyPage from './privacy';

describe('PrivacyPage', () => {
  it('renders', () => {
    const wrapper = shallow(<PrivacyPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
