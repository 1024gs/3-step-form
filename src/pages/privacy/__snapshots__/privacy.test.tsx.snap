// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPage renders 1`] = `
<div
  className="app"
>
  <Navbar />
  <br />
  <div
    className="container"
  >
    <FormNav />
    <br />
    <form
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <div
          className="form-check"
        >
          <input
            checked={true}
            className="form-check-input"
            id="updates"
            onChange={[Function]}
            type="checkbox"
            value="updates"
          />
          <label
            className="form-check-label"
            htmlFor="updates"
          >
            Receive updates about Tray.io product by email
          </label>
        </div>
        <div
          className="form-check"
        >
          <input
            checked={false}
            className="form-check-input"
            id="communication"
            onChange={[Function]}
            type="checkbox"
            value="communication"
          />
          <label
            className="form-check-label"
            htmlFor="communication"
          >
            Receive communication by email for other products created by the Tray.io team
          </label>
        </div>
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
